vocabularies/voc4cat/0007102.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007102>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007102"^^xsd:token ;
+    skos:definition "An archtype/ideal reactor type, in which a fluid is flowing through the pipe in a plug like manner. A reaction thus takes place over a certain distance in the reactor."@en ;
```

Comment:
If the pref. label contains the term "plug", you can't use the same term as part of the definition.

Reply:
is it acceptable if I have a short definition of "plug" after using it in the text? I don't really find a good alternative for plug and recommend synonyms also do not sound promising (bung, connection, cork, filling, fitting, occlusion, river, spigot, stopple, tampon, wedge; as can be found at https://www.thesaurus.com/browse/plug). Rephrasing everything so that I indirectly have the definition for plug inside the text might only make the definition and text unnecessarily difficult.